demisto_sdk/commands/common/hook_validations/dashboard.py

```diff
@@ -1,11 +1,108 @@
 from demisto_sdk.commands.common.hook_validations.base_validator import BaseValidator
+from demisto_sdk.commands.common.tools import print_error
 
 
 class DashboardValidator(BaseValidator):
+    @staticmethod
+    def get_widgets_from_dashboard(dashboard):
+        # type: () -> list
+        layout_of_dashboard: list = dashboard.get('layout', [])
+        widgets = []
+        if layout_of_dashboard:
+            widgets = [item.get('widget') for item in layout_of_dashboard]
+        return widgets
+
+    def is_valid_dashboard(self):
+        # type: () -> bool
+        """Check whether the dashboard is valid or not.
+
+        Returns:
+            bool. Whether the dashboard is valid or not
+        """
+        is_dashboard_valid = [
+            self.is_valid_version()
+        ]
+
+        # check only on added files
+        if not self.old_file:
+            is_dashboard_valid = all([
+                is_dashboard_valid,
+                self.is_id_equals_name()
+            ])
+
+        return is_dashboard_valid
+
     def is_valid_version(self):
+        # type: () -> bool
         """Return if version is valid. uses default method.
 
         Returns:
             True if version is valid, else False.
         """
         return self._is_valid_version()
+
+    def is_id_equals_name(self):
+        # type: () -> bool
+        """Check whether the dashboard ID is equal to its name.
+
+        Returns:
+            bool. Whether the file id equals to its name
+        """
+        return super(DashboardValidator, self)._is_id_equals_name('dashboard')
+
+    def is_not_excluding_fields(self):
```

Comment:
maybe change to "contains_forbidden_fields" or something like that?
"is_not_excluding" is confusing.